src/client/main.ts

```diff
@@ -66,6 +69,14 @@ export interface IntentResolution {
     version: string;
 }
 
+export type EventPayload = ChannelChangedPayload;
+export type Event = 'channel-changed';
```

Comment:
Can we rename `Event`, there's already a built-in `Event` type. Makes the `addEventListener` signature below look like it takes an event object.

Reply:
Changed to EventType